### PR TITLE
Add unsuccessful metrics to docs metrics

### PR DESCRIPTION
**Pull request by satk0**
Partially solves #295

### CODE REVIEW

File: scripts/add_metrics_to_readme.sh [hunk 1]
```diff
@@ -9,6 +9,27 @@ function getMetrics() {
     done
 }
 
+function failureConfigChange() {
+	local logstashCID=$(docker ps -a | grep 'logstash-exporter-logstash-1' | awk '{print $1}')
```
Comment (kuskoman) [inline scripts/add_metrics_to_readme.sh:13]: local logstashCID
logstashCID=$(...

File: scripts/add_metrics_to_readme.sh [hunk 2]
```diff
@@ -9,6 +9,27 @@ function getMetrics() {
...
+	# reload logstash
+	docker exec -it "$logstashCID" sh -c "kill -1 $logstashPID"
+
+	# bring back previous config 
+	# walk around with 'cp' to avoid replacing config inside container
+	docker exec "$logstashCID" sh -c "sed '\$d' $logstashConf > /tmp/prev_logstash.conf" 
+	docker exec "$logstashCID" sh -c "cp /tmp/prev_logstash.conf $logstashConf"
+
+	# reload logstash
+	docker exec -it "$logstashCID" sh -c "kill -1 $logstashPID"
```
Comment (kuskoman) [inline scripts/add_metrics_to_readme.sh:27]: logstash pid should be always 1 AFAIK

Comment (satk0) [reply scripts/add_metrics_to_readme.sh:30]: Okey, I will change it, but it's on you my friend if it doesn't work.

File: scripts/add_metrics_to_readme.sh [hunk 3]
```diff
@@ -9,6 +9,30 @@ function getMetrics() {
...
+	local logstashCID
+	logstashCID=$( docker ps -a | grep 'logstash-exporter-logstash-1' | awk '{print $1}' )
+	local logstashConf
+	logstashConf='/usr/share/logstash/pipeline/logstash.conf'
+
+	docker exec -it "$logstashCID" sh -c "echo 'Wrong Config' >> $logstashConf"
+
+	# reload logstash
+	# "logstash pid should be always 1 AFAIK" ~ Kuskoman, 2024
+	docker exec -it "$logstashCID" sh -c "kill -1 1"
```
Comment (kuskoman) [inline scripts/add_metrics_to_readme.sh:22]: assign 1 to a variable, so it will be more readable

**Review by kuskoman — CHANGES_REQUESTED** [review]

File: scripts/add_metrics_to_readme.sh (hunk 3 above)
Comment (satk0) [reply scripts/add_metrics_to_readme.sh]: Done 🫡

File: scripts/add_metrics_to_readme.sh [hunk 4]
```diff
@@ -9,6 +9,32 @@ function getMetrics() {
     done
 }
 
+function failureConfigChange() {
+	local logstashCID
+	logstashCID=$( docker ps -a | grep 'logstash-exporter-logstash-1' | awk '{print $1}' )
+	local logstashConf
+	logstashConf='/usr/share/logstash/pipeline/logstash.conf'
+	# "logstash pid should be always 1 AFAIK" ~ Kuskoman, 2024
```
Comment (kuskoman) [inline scripts/add_metrics_to_readme.sh:17]: :)

**Review by kuskoman — APPROVED** [review]

**Pull request merged**